examples/c++/README.md

```diff
@@ -5,5 +5,6 @@ for how to use StableHLO.
 
 Note: If you have a great example to highlight, we welcome contributions!
 
-* [stablehlo_add](./stablehlo_add.cpp): A simple example that demonstrates
-how to use the StableHLO library to add two numbers.
+* [stablehlo_add](./stablehlo_add.cpp): A simple example that demonstrates how to
+  * Use the StableHLO library to add two numbers.
+  * Interpret a StableHLO program using concrete inputs.
```

Comment:
can you add a code block here that shows an example invocation and output?

Reply:
Added.